README.md

```diff
@@ -57,12 +57,38 @@ This project supports a basic [Vagrant](http://docs.vagrantup.com/v2/getting-sta
 
 1. Run vagrant up command
 
-    vagrant up
+        vagrant up
+
 
 2. Visit [http://localhost:8085](http://localhost:8085) in your browser
 
 Look in [Vagrantfile](Vagrantfile) for configuration details.
 
+### Docker server
+
+This project supports a basic [Docker](https://www.docker.com/whatisdocker/) configuration
+
+1. [Install Docker](http://docs.docker.com/installation/)
+2. `cd` into the directory of the project
+3. Build the docker image based on the *Dockerfile*:
+        
+        docker build .
+
+4. Run `docker images` and find the *IMAGE ID* of the image
+5. Start the container (replace <image-id> with the Image ID found before):
+        
+        docker run -d -p 80:80 <image-id> /usr/sbin/apache2ctl -D FOREGROUND` 
```

Comment:
@pensiero I don't understand. Entry point allow pass arguments

Also CMD has an explicit example for apache2ctl https://docs.docker.com/articles/dockerfile_best-practices/#cmd